src/app/portlets/dot-starter/dot-starter.component.html

```diff
@@ -0,0 +1,240 @@
+<div class="p-grid dot-starter-top-content">
+    <div class="p-col-12">
+        <h2 class="dot-starter-title">{{ 'starter.title' | dm }}</h2>
+        <p class="dot-starter-description" [innerHTML]="'starter.description' | dm: [username]"></p>
+    </div>
+    <div class="p-col-12">
+        <div class="p-grid">
+            <div class="p-md-12 p-lg-7 p-grid dot-starter-top-main__section">
+                <a
```

Comment:
For example here, your `<a>` should be a 100% block that adjust to whatever the parent have.